services/storage/src/simcore_service_storage/s3_client.py

```diff
@@ -237,17 +242,22 @@ async def complete_multipart_upload(
         file_id: SimcoreS3FileID,
         upload_id: UploadID,
         uploaded_parts: list[UploadedPart],
+        sha256_checksum: SHA256Str | None,
     ) -> ETag:
-        response = await self.client.complete_multipart_upload(
-            Bucket=bucket,
-            Key=file_id,
-            UploadId=upload_id,
-            MultipartUpload={
+        inputs: dict[str, Any] = {
+            "Bucket": bucket,
+            "Key": file_id,
+            "UploadId": upload_id,
+            "MultipartUpload": {
                 "Parts": [
                     {"ETag": part.e_tag, "PartNumber": part.number}
                     for part in uploaded_parts
                 ]
             },
+            "ChecksumSHA256": None if sha256_checksum is None else str(sha256_checksum),
```

Comment:
MINOR: if `sha256_checksum` is already `None` or a `str`. there is no need to make an `if-else`